muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/locales/translations/en.json

```diff
@@ -1961,5 +1967,103 @@
       "image": "Image",
       "mathJax": "Math formula"
     }
+  },
+  "pedagogySupportPlan": {
+    "actions": {
+      "add": "$t(common:actions.add)",
+      "add_newRow": "Lisää uusi rivi",
```

Comment:
newRow --> row. Toisaalta myös on varmaan turha mainita "uusi rivi". Muistaakseni lokaalitestauksessa oli näitä "lisää uusi"-tilanteita, josta joku kommentoi, että "uusi" on turha sana tuossa välissä. Se ei oikeastaan sisällä mitään tärkeää tietoa.